src/extras/core/Path.js

```diff
@@ -151,7 +151,8 @@ THREE.Path.prototype = Object.assign( Object.create( THREE.CurvePath.prototype )
 			xRadius, yRadius,
 			aStartAngle, aEndAngle,
 			aClockwise,
-			aRotation || 0 // aRotation is optional.
+			aRotation || 0, // aRotation is optional.
+			this.curves.length
```

Comment:
yeah, `getPoints()` currently give you raw points on the outline of the Path.

I also agree that creating new Objects instance most performant here, but for the consistency of the code I agree with WestLangley here